### PR TITLE
Meta: Update CI workflow

### PR DESCRIPTION
**Pull request by fregante**
Follows https://github.com/npmhub/npmhub/pull/117 using https://github.com/fregante/ghatemplates#webextreleaseyml

### CODE REVIEW

File: .github/workflows/deploy-tag.yml [hunk 1]
```diff
@@ -1,53 +0,0 @@
...
-  deploy-to-chrome:
-    needs: build
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/download-artifact@v1
-      with:
-        name: artifact
-    - run: cd artifact && npx chrome-webstore-upload-cli@1 upload --auto-publish
-      env:
-        EXTENSION_ID: kgilejfahkjidpaclkepbdoeioeohfmj
```
Comment (fregante) [inline .github/workflows/deploy-tag.yml:38]: @tanmayrajani Can you set this as a repository secret? I don't have access to this repo’s settings

Comment (fregante) [reply .github/workflows/deploy-tag.yml:38]: Ping

Comment (fregante) [reply .github/workflows/deploy-tag.yml:38]: If you'd like to transfer the repo/admin to me I can add them myself

Comment (tanmayrajani) [reply .github/workflows/deploy-tag.yml:38]: I'm really sorry, I'm not sure why I'm seeing this for the first time :neutral_face: 
I just added `EXTENSION_ID` in the secrets

**Comment by fregante** [comment]
Thanks! 

Now releases can be done this way:

1. Visit the Actions tab
2. Select Release in the sidebar
3. Select Run workflow

Screen Shot 4

Releases are still "manual" (i.e. they're not scheduled) but they happen in a click.

The release notes will also be filled automatically. I filled the previous versions already: https://github.com/tanmayrajani/notifications-preview-github/releases

**Comment by tanmayrajani** [comment]
Thank you so much 🙌

**Pull request merged**